lua/lspconfig/configs/regal.lua

```diff
@@ -8,6 +8,28 @@ return {
       return util.root_pattern '*.rego'(fname) or util.find_git_ancestor(fname)
     end,
     single_file_support = true,
+    handlers = {
+      ['regal/startDebugging'] = function(_, result)
```

Comment:
just `vim.lsp.handler[method] = function(_, result)` since we plan to remove config.handlers in future.

Reply:
I didn't know about that. Thanks.